stories/02-features/05-Style-with-themes.stories.mdx

```diff
@@ -0,0 +1,71 @@
+<Meta title="Features|Style with Themes" />
+
+# Theming a Styled Component
+
+An assortment of minimally Styled Components is included in Irving' styled-components package. Styles are easily overwritten by providing Irving with your own custom themes. Each styled-component's configuration includes a theme property that maps to the default style included in the component package. For example, the default.js theme for the container component is:
+
+```js
+import styled from 'styled-components';
+
+/* eslint-disable import/prefer-default-export */
+export const ContainerWrapper = styled.div`
+  margin: auto;
+  width: 100%;
+`;
+/* eslint-enable */
+
+```
+
+The conditions needed to apply themes are:
+* a `themes` folder must be created in the irving directory with an `index.js` file within. It's a good practice to organize your themes by component name, so to style a container, for example, create a `container` folder within the `themes` folder.
```

Comment:
I think it might help to include the path to the irving directory here, and err on the side of overcommunication. I also really love that you've provided an example of the folder structure below!

Reply:
check the added copy

Reply:
nice addition!